src/graphql/root/mutation/user-update-username.ts

```diff
@@ -35,7 +35,9 @@ const UserUpdateUsernameMutation = GT.Field({
     if (result instanceof Error) {
       return {
         errors: [mapAndParseErrorForGqlResponse(result)],
-        ...(result instanceof UsernameIsImmutableError ? { user: domainUser } : {}),
+
+        // FIXME: what is this return for?
```

Comment:
Do we still need this `FIXME` comment?

Reply:
yeah I think the line above is not used/useful, but it's probably better to delete in another segregated PR